src/server/handlers/manager/join.rs

```diff
@@ -52,7 +52,7 @@ impl JsonRPCRequestHandler for JoinSubnetHandler {
         let collateral = handlers::f64_to_token_amount(request.collateral)?;
 
         let subnet_config = conn.subnet();
-        check_subnet(subnet_config)?;
+        // check_subnet(subnet_config)?;
```

Comment:
We should still check the parent (I guess you commented it due to the Subnet change)

Reply:
yeah, I would say the checking needs to be updated as well. EVM and FVM checking should be different.